scripts/FilterAFComparisonTable.wdl

```diff
@@ -0,0 +1,77 @@
+version 1.0
```

Comment:
Can this be run as a single workflow or does it need to be broken into 3 workflows?